content/applications/general/iot.rst

```diff
@@ -1,14 +1,75 @@
-:nosearch:
 :show-content:
 :hide-page-toc:
-:show-toc:
+:nosearch:
 
 ========================
 Internet of Things (IoT)
 ========================
 
+Odoo IoT allows to connect physical devices such as barcode scanners, receipt printers, payment
+terminals, measurement tools, etc. to an Odoo database using an IoT system.
+
+The following IoT systems are supported:
+
+- :doc:`IoT box <iot/iot_box>`: micro-computer, plug-and-play device (IoT program pre-installed);
+- :doc:`Windows virtual IoT <iot/windows_iot>`: Windows program that can be installed on a Windows
+  computer.
+
+.. important::
+   An IoT box subscription is required to use IoT systems with a secured connection.
+
+.. note::
+   - :abbr:`MRP (Material Requirement Planning)` devices, including cameras and measurement tools,
+     are not compatible with Windows virtual IoT.
+   - You can use multiple IoT systems at the same time.
+   - It is also possible to create a Windows Virtual Machine on a MacOS/Linux computer. However,
+     this option is not supported by Odoo, and no troubleshooting assistance will be provided.
+
+.. cards::
+
+   .. card:: IoT box
+      :target: iot/iot_box
+      :large:
+
+      Set up an IoT box.
+
+   .. card:: Windows virtual IoT
+      :target: iot/windows_iot
+      :large:
+
+      Set up Windows virtual IoT.
+
+   .. card:: IoT system connection to Odoo
+      :target: iot/connect
+
+      Connect the IoT system to your Odoo database and troubleshoot potential connection issues.
+
+   .. card:: Devices
+      :target: iot/devices
+
+      Connect devices such as printers, screens, measurement tools, etc., to the IoT system.
+
+   .. card:: HTTPS certificate
+      :target: iot/iot_advanced/https_certificate_iot
+
+      Verify your IoT system and database meet the eligibility requirements for HTTPS certificate
+      generation and address any potential issues.
+
+   .. card:: IoT system updates
+      :target: iot/iot_advanced/updating_iot
+
+      Update your IoT system's image, core code, and handlers.
```

Comment:
do we need a "," before an and ?

also, maybe precise here keyword like "reset" "factory reset" that might be useful to describe this section. Same for "benefit from IoT latest fix and feature"